pkg/apis/serving/k8s_validation_test.go

```diff
@@ -419,7 +422,7 @@ func TestPodSpecMultiContainerValidation(t *testing.T) {
 				}},
 			}},
 		},
-		cfgOpts: []configOption{withPodSpecFieldRefEnabled()},
+		cfgOpts: []configOption{withPodSpecFieldRefEnabled(), withMultiContainerDisabled()},
```

Comment:
I think its not required to add `withMultiContainerDisabled()` here because the testcase has single container so it won't go to [default case](https://github.com/knative/serving/blob/master/pkg/apis/serving/k8s_validation.go#L287-L289).
So we can remove that